src/ethereum_test_tools/spec/state/types.py

```diff
@@ -44,49 +43,63 @@ class FixtureEnvironment:
         default=100000000000000000,
         json_encoder=JSONEncoder.Field(
             name="currentGasLimit",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     number: NumberConvertible = field(
         default=1,
         json_encoder=JSONEncoder.Field(
             name="currentNumber",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     timestamp: NumberConvertible = field(
         default=1000,
         json_encoder=JSONEncoder.Field(
             name="currentTimestamp",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     prev_randao: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
             name="currentRandom",
-            cast_type=Number,
+            cast_type=Hash,
         ),
     )
     difficulty: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
             name="currentDifficulty",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     base_fee: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
             name="currentBaseFee",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     excess_blob_gas: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
-            name="currentExcessBlobGas",
-            cast_type=Number,
+            name="parentExcessBlobGas",
```

Comment:
I think this should be `currentExcessBlobGas`, because adding the `parentExcessBlobGas`, apart from it requiring a `parentBlobGasUsed`, requires a block header computation, which I think is outside of the scope of a state test.

By using `currentExcessBlobGas`, the state test consumer can directly know how much the executing transaction must be charged with per blob.